### PR TITLE
Homepagestyle

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
Hey Bookis, I made some edits to this pull request but everything should be good to go. Would love to hear any feedback/changes that you think could improve before merging!

**Comment by bookis** [comment]
Will you add `.sass-cache` to the `.gitignore`. Then run `git rm -rf .sass-cache`, commit, and push to permanently delete them. There are like 100 files that are just temporary files that we don't need in the repo.

File: index.html [hunk 1]
```diff
@@ -6,11 +6,13 @@
   </head>
   <body>
     <header>
-      <span class='brand'>Gmail Sender</span>
       <ul class='nav' role='nav'>
         <li><a href="http://www.scheduled.email/auth/google_oauth2/new?redirect_to=http://localhost:8080">Sign In</a>
         <li><a href="http://www.scheduled.email/sign-out?redirect_to=http://localhost:8080" class='new-window'>Sign Out</a>
         <li><a href="#new-schedule" class="new-schedule">+ Schedule</a></li>
+        <div class = "logo">
```
Comment (bookis) [inline index.html]: this `div` should be within the `li`. Within a `ul` it's direct children should only be `li` elements, within those `li` elements you can have other stuff, like a `div`. Although, with this case you could get rid of the `div` completely and just put the `class="logo"` on the `li`.

File: index.html [hunk 2]
```diff
@@ -21,51 +23,94 @@
...
+          Body: Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed
+          do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+          ad minim veniam, quis nostrud exercitation ullamco laboris nisi
+          ut aliquip ex ea commodo consequat. Duis aute irure dolor in
+          reprehenderit in voluptate velit esse cillum dolore eu fugiat
+          nulla pariatur. Excepteur sint occaecat cupidatat non proident,
+          sunt in culpa qui officia deserunt mollit anim id est laborum.
+          </p>
+      </div>
+      <div>
```
Comment (bookis) [inline index.html]: It looks like this is just an empty `div`, will you remove it just to keep things clean.

File: src/scss/main.scss [hunk 3]
```diff
@@ -4,3 +4,176 @@
...
+  background-color:#80D8FF;
+  color:white;
+  width:300px;
+  height:70px;
+  text-align:center;
+  padding-top:20px;
+  font-size:20px;
+}
+
+body, html{
```
Comment (bookis) [inline src/scss/main.scss]: Just for code style; I like to keep my really general selectors (like `body` and `html` at the very top of the file, with more specific selectors like `.headline` below.

File: src/scss/main.scss [hunk 4]
```diff
@@ -4,3 +4,176 @@
...
+
+form{
+  border-left:1px solid black;
+  border-right:1px solid black;
+  border-bottom:1px solid black;
+  margin:50px;
+  height:800px;
+}
+
+form ul{
```
Comment (bookis) [inline src/scss/main.scss]: Since the `ul` was removed from the `form`, this rule can be removed.

File: src/scss/main.scss [hunk 5]
```diff
@@ -4,3 +4,176 @@
...
+}
+
+.schedule-send input{
+  width:100%;
+  height:40px;
+}
+.schedule-send li{
+  list-style-type:none;
+}
+.create-email h2{
```
Comment (bookis) [inline src/scss/main.scss]: For organization, I like to keep similar selectors near each other, there is a `.create-email` selector on line 64, I would move this up to be right below, so they are grouped together.

**Comment by bookis** [comment]
I left a bunch of nit-picky notes, but I think important for the long term maintenance. Let me know if you have any questions.

**Comment by ghost** [comment]
Ok Great! I fixed the files and let me know if it looks ok on your end?

**Comment by bookis** [comment]
Did you do the `git rm -rf .sass-cache`, commit, and push? It looks like the files are still there.

**Comment by bookis** [comment]
Thanks! Looks great.

**Pull request merged**